oidc-service-impl/src/main/java/io/mosip/esignet/services/AuthorizationHelperService.java

```diff
@@ -244,6 +252,8 @@ protected KycAuthResult handleInternalAuthenticateRequest(@NonNull AuthChallenge
             }
             log.error("ID token in the challenge is not matching the required conditions. isCookiePresent: {}, isHaltedTransactionFound: {}",
                     result.isPresent(), haltedTransaction!=null);
+        } catch (EsignetException e) {
+            throw new EsignetException(INVALID_INDIVIDUAL_ID);
```

Comment:
What is the need to create new instance? We could throw the same "e" object

Reply:
changed this